spec/models/comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+# Tests Comments
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validations' do
+    user = User.create(name: 'Rishi', photo: 'Image will be displayed here', bio: 'Hello pals', posts_counter: 0)
+    post = Post.create(title: 'First Post', text: 'This is my first post', author_id: user.id, comments_counter: 0,
+                       likes_counter: 0)
+    subject { described_class.create(text: 'This is my first comment', post:, user:) }
```

Comment:
- [ ] Here Also, Please add a test for the `update_comment_count` method inside your `Comment` model. We need to make sure all custom methods are tested.